src/util/sock.cpp

```diff
@@ -51,15 +51,21 @@ Sock& Sock::operator=(Sock&& other)
 
 SOCKET Sock::Get() const { return m_socket; }
 
-SOCKET Sock::Release()
-{
-    const SOCKET s = m_socket;
+void Sock::Reset() {
```

Comment:
`Reset` is a strange API function for a socket. Conceptually, I like `Close` better.

Reply:
Removed!